invenio_rdm_migrator/load/postgresql.py

```diff
@@ -137,6 +137,11 @@ def _generate_rows(self, **kwargs):
         """Yield generated rows."""
         pass
 
+    @abstractmethod
```

Comment:
minor: not all tables might have references, so maybe having a return default in the interface?

Reply:
I was thinking that...ok I will do it!